js/components/Textcolor.js

```diff
@@ -0,0 +1,12 @@
+const React = require("react");
+
+class Textcolor extends React.Component {
+	render() {
+		return (
+			<font size="3" color="red">{this.props.children}</font>
```

Comment:
The color (and maybe size!) properties shouldn't be hard-coded, but should use properties.